OxfordBodleian/Ullendorff/BDLeu2.xml

```diff
@@ -34,9 +37,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <title ref="LIT2246salotz" type="complete"/>
+                            <incipit xml:lang="gez">
+                                <locus from="3r" to="3v"/>
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ዛ</hi>ቲ፡ ጸሎት፡ ዘእግዝእትነ፡
+                                <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ፡ እሙ፡ 
+                                <hi rend="rubric">ለብርሃን፡ ቅድስት፡ ድንግል፡ ወንጽሕት፡ ዘጸሐፈ፡ አብር</hi>ኮሮስ፡ <sic>ረድዕ፡</sic> ዮሐንስ፡ 
+                                <seg type="supplication">ጸሎታ፡ ወበረከታ፡ የሀሉ፡ ምስለ፡ ገብራ፡ 
+                                    <persName ref="PRS13999WaldaM"><hi rend="rubric">ወልደ፡ ሚካኤል፡</hi></persName> ለዓለመ፡ ዓለም፡
+                                    አሜን<hi rend="rubric" rendition="#partialRubric">፨</hi></seg>
+                                ዘጸ<pb n="3v"/>ለየት፡ ባቲ፡ አመ፡ 
+                                <hi rend="rubric" rendition="#partialRubric">፳</hi>ወ<hi rend="rubric" rendition="#partialRubric">፩</hi>
+                                ለወርኃ፡ ሰኔ፡ በደብረ፡ ጎልጎታ፡ ዝውእቱ፡ መቃብረ፡ እግዚእነ፡ 
+                                <hi rend="rubric">ኢየሱስ፡ ክርስቶስ፡</hi> ዘሎቱ፡ <sic>ስሐት፡</sic> እስከ፡ ለዓለመ፡ ዓለም፡ አሜን<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                            </incipit>
+                            <explicit xml:lang="gez" type="supplication">
+                                <locus from="39r" to="39v"/>
+                                ወትብል፡ ስብሐት፡ ለከ፡ አኰቴት፡ ለመንግሥትከ<hi rend="rubric" rendition="#partialRubric">፨</hi> ወስብሐት፡ ለአቡከ፡ ኄር፡
+                                ሰማያዊ፡ ወመንፈስ፡ ቅዱስ፡ ማኅየዊ<hi rend="rubric" rendition="#partialRubric">፨</hi> ይእዜኒ፡ ወዘልፈኒ፡ ለዓለመ፡ ዓለም፡ አሜን፡
+                                <pb n="39v"/> አሜን፡ ለይኩን፡ ለይኩን፡
+                              <seg type="supplication">ኦእግዝእትየ፡ <hi rend="rubric">ማርያም፡</hi> በዝንቱ፡ ጸሎትኪ፡ ወበዝንቱ፡ ቃለ፡ ኪዳንኪ፡ ዕቀበኒ፡ ወአድኅንኒ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ሊተ፡ ለኃጥእ፡ 
+                                <sic>ገብርከ፡</sic> 
+                                <persName ref="PRS13999WaldaM"><hi rend="rubric">ወልደ፡</hi> 
+                                <add place="overstrike">ም<gap reason="illegible" unit="chars" quantity="1"/>ር<gap reason="illegible" unit="chars" quantity="1"/></add> <hi rend="rubric">ንግዳ፡ ብሩ።</hi></persName>
+                                  በ<date calendar="grace" when-custom="1947" notBefore="1954" notAfter="1955"><hi rend="rubric" rendition="#partialRubric">፲</hi>ወ<hi rend="rubric" rendition="#partialRubric">፱</hi>፡ 
+                                      <hi rend="rubric" rendition="#partialRubric">፻</hi><hi rend="rubric" rendition="#partialRubric">፵</hi>ወ<hi rend="rubric" rendition="#partialRubric">፯</hi> 
+                                      ዓመተ፡ ምሕረት<hi rend="rubric" rendition="#partialRubric">፨</hi></date></seg> 
+                                ወጸሐፊሁ፡ <persName ref="PRS14000WaldaA" role="scribe">አባ፡ ወልደ፡ አረጋዊ።</persName> <foreign xml:lang="am">ጠያቂ፡</foreign> <persName ref="PRS14001GabraI">ገብረ። ኢየሱስ።</persName>
```

Comment:
What about `<persName ref="PRS14000WaldaA" role="scribe"><roleName type="title">አባ፡</roleName> ወልደ፡ አረጋዊ።</persName>`  with roleName?
About ጠያቂ: this Ge'ez word means - as an adjective - "acute, insightful, scrupulous", but the way the formula ጠያቂ፡ ገብረ። ኢየሱስ። parallels አባ፡ ወልደ፡ አረጋዊ። allows to consider ጠያቂ as a title as well, like አባ፡. I see that it also exists in Amharic (Kane, vol. II, p. 2174) but the meaning is not satisfactory. Perhaps Solomon can say something on this? Perhaps the word has a regional or local connotation and this might give a clue about its provenance

Reply:
Addisie suggests that it might describe here the commissioner, as the "one who requests". I find that quite convincing in the context. Let's see if Jonas has some ideas as well.

Reply:
That's an interesting proposal! I also think that we can accept it, perhaps followed by a "(?)".